internal/controllers/successors_test.go

```diff
@@ -1,7 +1,8 @@
-package variablesources_test
+package controllers_test
```

Comment:
It might be easier to review this file with "hide whitespace" option (`-w` in `git diff`).